warehouse/models/intermediate/gtfs/gtfs_quality/_int_gtfs_quality.yml

```diff
@@ -23,29 +23,23 @@ models:
             - severity
 
   - name: int_gtfs_quality__no_schedule_validation_errors
-    tests:
+    tests: &stg_gtfs_guideline_tests
       - dbt_utils.unique_combination_of_columns:
           combination_of_columns:
             - date
             - feed_key
       - dbt_utils.equal_rowcount:
           compare_model: ref('int_gtfs_quality__schedule_feed_guideline_index')
-    columns:
+    columns: &stg_gtfs_guideline_columns
```

Comment:
I think you made this change in your other new shape branch too, just flagging in case the merges get weird

Reply:
yeah - in hindsight I should have done just one branch for all 3 changes. merging will be fun